.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules/*
+
+/text.md
```

Comment:
- [ ] I know you have exhausted a lot of reviews but please try to implement everything I point out to you. You only have one submission left so before you resubmit, make sure you've completed everything :+1: and please don't delete the branch so the next reviewer can see what I asked you to do and check if you did them and approve you. :100: 
- [ ] Please remove the forward slash `/` in front of `text.md`. This file is supposed to be ignored by git but because of the slash you put in front of it it didn't find the file so it wasn't ignored that's why it is part of the files you have on GitHub but node modules aren't. :+1: